assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -213,19 +213,19 @@
 }
 
 .googlesitekit-settings-notice-adsense-top-earning-pages-widget {
-	align-items: center;
-	flex-wrap: wrap;
+	align-items: normal;
+	flex-flow: column wrap;
 	padding-bottom: 24px;
 	padding-top: 24px;
 	row-gap: 15px;
```

Comment:
Hey @tofumatt, this is looking good, the only real issue I can see is that the CTA can get too close to the text at some viewport widths, for example here at `974px`:

![image](https://github.com/google/site-kit-wp/assets/18395600/70373a3c-4072-4eac-9c33-5a0817922034)

Having taken a quick look, I think a reasonable way to address this would be to update this `row-gap` property to be a `gap` property that covers both rows and columns. If we set `gap: 15px` and then drop the `margin-right` for `.googlesitekit-settings-notice__icon` down to `5px` it should give a decent effect.


```suggestion
	gap: 15px;
```